src/main/java/com/github/igotyou/FactoryMod/recipes/PrintNoteRecipe.java

```diff
@@ -93,21 +88,36 @@ public void applyEffect(Inventory i, FurnCraftChestFactory fccf) {
 		logAfterRecipeRun(i, fccf);
 	}
 	
-	private void addTextLines(ItemStack printingPlateStack, List<String> result) {
+	private BookInfo getBookInfo(ItemStack printingPlateStack) {
 		ItemStack book = createBook(printingPlateStack, 1);
 		BookMeta bookMeta = (BookMeta)book.getItemMeta();
 		String text = bookMeta.getPageCount() > 0 ? bookMeta.getPage(1): "";
-		String[] lines = text.split("§0\n");
+		String[] lines = text.split("\n");
+		List<String> fixedLines = new ArrayList<String>();
 		
-		for(int i = 0; i < lines.length; i++) {
-			String line = lines[i];
+		for(String line : lines) {
+			String fixedLine = line.replaceAll("§0", ChatColor.GRAY.toString());
```

Comment:
Oh yes, absolutely.
I will replace it by embedded constant and retest later today.